nest/command_line.py

```diff
@@ -334,43 +391,11 @@ def _identity(x):
                 print('%0.1f' % display_temp(target))
 
         elif cmd == 'show':
-            # TODO should pad key? old code put : out 35
-            print('Device: %s' % device.name)
-            print('Where: %s' % device.where)
-            print('Away                  : %s' % device.structure.away)
-            print('Mode                  : %s' % device.mode)
-            print('State                 : %s' % device.hvac_state)
-            print('Can Heat              : %s' % device.can_heat)
-            print('Can Cool              : %s' % device.can_cool)
-            print('Has Humidifier        : %s' % device.has_humidifier)
-            print('Has Dehumidifier      : %s' % device.has_humidifier)
-            print('Has Fan               : %s' % device.has_fan)
-            if device.has_fan:
-                print('Fan                   : %s' % device.fan)
-                print('Fan Timer             : %s' % device.fan_timer)
-            print('Has Hot Water Control : %s' % device.has_hot_water_control)
-            if device.has_hot_water_control:
-                print('Hot Water Temp        : %s' % device.fan)
-            print('Temp                  : %0.1f%s' % (device.temperature,
-                  device.temperature_scale))
-            helpers.print_if('Humidity              : %0.1f%%',
-                             device.humidity)
-            if isinstance(device.target, tuple):
-                print('Target                 : %0.1f-%0.1f%s' % (
-                    display_temp(device.target[0]),
-                    display_temp(device.target[1]),
-                    device.temperature_scale))
-            else:
-                print('Target                : %0.1f%s' %
-                      (display_temp(device.target), device.temperature_scale))
-
-            helpers.print_if('Away Heat             : %0.1fC',
-                             device.eco_temperature[0])
-
-            helpers.print_if('Away Cool             : %0.1fC',
-                             device.eco_temperature[1])
-
-            print('Has Leaf              : %s' % device.has_leaf)
+            count = 0
```

Comment:
If this is not directly related to the streaming API, please submit changes into a pull request in the main repo.